platforms/extensibility/plugin-development/src/integTest/groovy/org/gradle/plugin/devel/impldeps/GradleImplDepsConcurrencyIntegrationTest.groovy

```diff
@@ -20,7 +20,7 @@ import org.gradle.api.Plugin
 import org.gradle.test.precondition.Requires
 import org.gradle.test.preconditions.UnitTestPreconditions
 
-@Requires(UnitTestPreconditions.HighPerformance)
+//@Requires(UnitTestPreconditions.HighPerformance)
 class GradleImplDepsConcurrencyIntegrationTest extends BaseGradleImplDepsIntegrationTest {
```

Comment:
What's HighPerformance here mean?

Reply:
Ahh I knew this made its way into one of my PRs. Forgot which.

It means non-mac. Because apparently our CI mac machines are not fast.

Will revert this.